Makefile.checker

```diff
@@ -73,9 +73,13 @@ checker/%.cmx: checker/%.ml
 	$(SHOW)'OCAMLOPT  $<'
 	$(HIDE)$(OCAMLOPT) $(CHKLIBS) $(OPTFLAGS) -c $<
 
-md5chk:
+tools/md5sum.native: tools/md5sum.ml
```

Comment:
`Digest` is C code: https://github.com/ocaml/ocaml/blob/trunk/stdlib/digest.ml#L24
Compiling to native code makes no difference here, and forces the host of the compilation to have a native backend.
I think you should compile to byte code.

Reply:
In fact why even compile?

Reply:
> In fact why even compile?

Easy. It makes the patch smaller: Nothing in the Coq code base is using the interpreter yet so I wanted to avoid problems like having two tools (ocamlfind & ocaml) that are not from the same environment, having two different Makefile variables and so on.

Anyway, I'll change to bytecode.

Reply:
Well the configure script uses the interpreter, but I can believe it's not used in makefiles.

Reply:
Ok, fair. It is set by the configure script but not used anywhere-else yet I believe.